README.md

```diff
@@ -26,18 +26,21 @@ Using the example certificate chain and CA files in this repo:
      CA-1: Subject:  CN=DST Root CA X3,O=Digital Signature Trust Co.
            Subject Key Identifier:        c4:a7:b1:a4:7b:2c:71:fa:db:e1:4b:90:75:ff:c4:15:60:85:89:10
            Issuer:   CN=DST Root CA X3,O=Digital Signature Trust Co.
+           VALID:    2023-03-03 16:00:00+00:00 <-> 2028-03-03 16:00:00+00:00
```

Comment:
Two requests with regard to:  'Valid' / 'INVALID'

1. Place the validity line between Subject and SKI to emphasise that it relates to the subject and not the issuer.
2. Normal-case "Valid" so it doesn't stand out as abnormal.  Leave "INVALID" as all caps.

Thanks,
Eric

Reply:
done